pennylane/debugging.py

```diff
@@ -241,13 +366,53 @@ def state():
 
 
 def expval(op):
-    """Compute the expectation value of an observable.
+    """Compute the expectation value of an observable at the
+    current state of the quantum circuit.
```

Comment:
This is consistent with the docs for the other functions:
```suggestion
    """Compute the expectation value of an observable at the
    current point in the quantum circuit.
```